<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" tests="6" skipped="0" failures="0" errors="0" timestamp="2020-11-25T14:38:59" hostname="lichangbaodeMacBook-Pro.local" time="0.156">
  <properties/>
  <testcase name="clearFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.073"/>
  <testcase name="getFailPutExceptionFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.018"/>
  <testcase name="evictFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.017"/>
  <testcase name="putFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.015"/>
  <testcase name="getFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.016"/>
  <testcase name="getPutNewElementFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.017"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
